src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from "prop-types";
+import React from "react";
+import s from "./BlockQuoteBlock.module.css";
+
+const BlockQuoteBlock = ({ quote, author }) => {
+  const GridAreaMiddle = () => (
+    <div className={s.gridAreaMiddle}>
+      <div className={s.quote}>{quote}</div>
+      <br />
```

Comment:
You shouldn't use `<br />` tags for spacing purposes. Instead, you should just use CSS.

Reply:
👍 I used margin-bottom in CSS for line separation.